pom.xml

```diff
@@ -157,6 +157,10 @@
             <artifactId>spring-cloud-stream-binder-rabbit</artifactId>
             <scope>runtime</scope>
         </dependency>
+        <dependency>
+            <groupId>io.micrometer</groupId>
+            <artifactId>micrometer-registry-prometheus</artifactId>
+        </dependency>
```

Comment:
I think you'd rather import spring-boot-actuator as compiled dependency than registry-prometheus. Spring-boot-actuator embed the API and registry-prometheus is the implementation for prometheus.

Reply:
Done